packages/application-shell/src/components/setup-flop-flip-provider/setup-flop-flip-provider.tsx

```diff
@@ -17,6 +17,40 @@ import type {
 } from '../../types/generated/mc';
 import AllFeaturesQuery from './fetch-all-features.mc.graphql';
 
+const WAIT_FOR_REMOTE_FLAGS_TIMEOUT = 500; // milliseconds
```

Comment:
This is the maximum time we will wait for remote flags to be resolved. 
After that, we will render the application anyway (with the default flags values) and, if the flags resolution happens after this time threshold, we ignore it.